helpers/exclusion_list.txt

```diff
@@ -23,6 +23,7 @@
 ./tools/ml-dataprep
 ./tools/ml-auto-eda
 ./tools/netblock-monitor
+./tools/service-account-provider/example_gitlab_pipeline
```

Comment:
Why did you add this folder to the exclusion list ? Is the linting not passing ?

Reply:
There are a few environment variables that are not set within the shell script (they are set from within the GitLab pipeline), and the linter was complaining about them. Since the script is just an example for how to use the tool, I added the exclusion.
Although it would be great if you have a suggestion how I could handle that differently.